### PR TITLE
More tests to shake out some cases in the json parsing

### PR DESCRIPTION
**Pull request by thomasvl**

### CODE REVIEW

**Comment by thomasvl** [comment]
@xfxyjwf - fyi.

File: conformance/conformance_test.cc [hunk 1]
```diff
@@ -771,13 +775,60 @@ bool ConformanceTestSuite::RunSuite(ConformanceTestRunner* runner,
   ExpectParseFailureForJson(
       "TrailingCommaInAnObject",
       R"({"fieldname1":1,})");
+  ExpectParseFailureForJson(
+      "TrailingCommaInAnObjectWithSpace",
+      R"({"fieldname1":1 ,})");
```
Comment (xfxyjwf) [inline conformance/conformance_test.cc]: Are there any implementations passing "TrailingCommaInAnObject" but failing this?

**Comment by xfxyjwf** [comment]
LGTM

File: conformance/conformance_test.cc (hunk 1 above)
Comment (thomasvl) [inline conformance/conformance_test.cc]: Yes, C++

**Pull request merged**